pages/linux/serialver.md

```diff
@@ -0,0 +1,17 @@
+# serialver
+
+> Returns the serialVersionUID of classes.
+> It does not set a security manager by default.
+> More information: <https://manned.org/serialver>.
```

Comment:
```suggestion
> More information: <https://docs.oracle.com/en/java/javase/18/docs/specs/man/serialver.html>.
```
Update the link to official documentation, also this page must be placed in `common`.